aio/aio-proxy/aio_proxy/search/helpers/service_public.py

```diff
@@ -0,0 +1,29 @@
+from aio_proxy.labels.helpers import (
+    natures_juridiques,
+    natures_juridiques_service_public,
+)
+
+
```

Comment:
I dont get that file, nor why you use a dedicated `natures_juridiques_service_public`rather than a condition on nature juridique codes

Reply:
Does it handle the cas where you have a nature juridique filter and a est_service_public ?

Reply:
And in that case do you choose to do a OR or a AND ?

Reply:
it does handle it and it uses an `and` logic, which is the logic we implement with all filters.
If the `nature_juridique` indicated does not match that on a `service_public`, then the results would be empty.

Reply:
I went with your solution anyways as it is easier to implement and to understand.